docs/usage.md

```diff
@@ -119,6 +119,22 @@ To commit all the tracked changes in base repo and any enabled modules:
 evg-module-manager git-commit --commit-message "my commit message"
 ```
 
+## Creating github pull request
+
+Use `pull-request` command to create pull requests across base repo and all enabled modules.
+In order to fully utilize this function, [github CLI](https://cli.github.com/) need to be installed
+in your system. After the installation, you need to give authorization to the github CLI, by issuing
+[gh auth login](https://cli.github.com/manual/gh_auth_login). Github CLI would prompt interactive command
+line and errors.
+
+After local changes has committed in all repos, you need to push your branch to the remote before use the pull request
```

Comment:
Why does the user need to push their branch. That seems like something the tool should be doing.